build-system/tasks/bundle-size.js

```diff
@@ -122,6 +157,30 @@ function storeBundleSize(bundleSize) {
   });
 }
 
+function compareBundleSize(maxBundleSize) {
+  const cmd = `npx bundlesize -f "${runtimeFile}" -s "${maxBundleSize}"`;
+  log('Running ' + cyan(cmd) + '...');
+  const output = getStdout(cmd).trim();
+
+  const error = output.match(/ERROR .*/);
+  if (error || output.length == 0) {
+    return [error || '[no output from npx command]', STATUS_ERROR, ''];
+  }
+
+  const bundleSizeFormatMatches = output.match(/: (\d+.?\d*KB)/);
+  if (bundleSizeFormatMatches) {
+    const pass = output.match(/PASS .*/);
+    const fail = output.match(/FAIL .*/);
```

Comment:
Minor: You should be able to do this in 1 instead of 3 regexes.

Reply:
Done